src/theme/button.scss

```diff
@@ -0,0 +1,8 @@
+
+button {
+    appearance: none;
+    background-color: var(--theme-color--button);
+    color: var(--theme-color--ink);
+    padding: var(--theme-spacing--2);
+    text-transform: uppercase;
```

Comment:
Try and make this button looks as much like the design provided as possible. Replicating designs into CSS is a key front-end developer skill.